packages/integrations/test/captureconsole.test.ts

```diff
@@ -81,9 +81,9 @@ describe('CaptureConsole setup', () => {
       expect(global.console.error).not.toBe(originalConsole.error);
       expect(global.console.log).not.toBe(originalConsole.log);
       expect(global.console.assert).not.toBe(originalConsole.assert);
+      expect(global.console.trace).not.toBe(originalConsole.trace);
```

Comment:
As a result of adding `trace` to `CONSOLE_LEVELS`, we are now monkeypatching everything but `console.table`.

I am not sure whether we want this or not. I would appreciate someone double-checking this.

Also, why are we not monkeypathcing `console.table` as well?

Reply:
Yeah we want this. Don't know why we're not using table, but there are a few things to consider: 1. it has sort of a different API than the other methods 2. No IE11 support